### PR TITLE
[FSTORE-1480] Deduplicate git*

### PR DESCRIPTION
**Pull request by aversey**
Requires https://github.com/logicalclocks/hopsworks-api/pull/220 and https://github.com/logicalclocks/hopsworks-api/pull/245 to be merged first.

JIRA Issue: [FSTORE-1480](https://hopsworks.atlassian.net/browse/FSTORE-1480)

Priority for Review: -

Related PRs: -

**How Has This Been Tested?**

- [ ] Unit Tests
- [ ] Integration Tests
- [ ] Manual Tests on VM

**Checklist For The Assigned Reviewer:**

```
- [ ] Checked if merge conflicts with master exist
- [ ] Checked if stylechecks for Java and Python pass
- [ ] Checked if all docstrings were added and/or updated appropriately
- [ ] Ran spellcheck on docstring
- [ ] Checked if guides & concepts need to be updated
- [ ] Checked if naming conventions for parameters and variables were followed
- [ ] Checked if private methods are properly declared and used
- [ ] Checked if hard-to-understand areas of code are commented
- [ ] Checked if tests are effective
- [ ] Built and deployed changes on dev VM and tested manually
- [x] (Checked if all type annotations were added and/or updated appropriately)
```

[FSTORE-1480]: https://hopsworks.atlassian.net/browse/FSTORE-1480?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: python/hopsworks/git_commit.py [hunk 1]
```diff
@@ -17,7 +17,7 @@
 import json
 
 import humps
-from hopsworks import util
+from hopsworks_common import util
 
 
 class GitCommit:
```
Comment (robzor92) [inline python/hopsworks/git_commit.py:23]: This file looks wrong, should it not be importing `git_commit `from `hopsworks_common`?

**Review by robzor92 — CHANGES_REQUESTED** [review]
Also the github mkdocs action is failing

**Comment by aversey** [comment]
> Also the github mkdocs action is failing

It was, since this PR depends on previous ones. Now that they are merged, the mkdocs error will go away too.

File: python/hopsworks/git_commit.py (hunk 1 above)
Comment (aversey) [reply python/hopsworks/git_commit.py:23]: Yes, for sure. Thanks for catching it!

**Review by robzor92 — APPROVED** [review]

**Pull request merged**